spec/pom.xml

```diff
@@ -17,74 +17,88 @@
 
 -->
 
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
     <parent>
         <groupId>org.eclipse.ee4j</groupId>
         <artifactId>project</artifactId>
         <version>1.0.6</version>
         <relativePath/>
     </parent>
-    <modelVersion>4.0.0</modelVersion>
+
     <groupId>jakarta.xml.bind</groupId>
     <artifactId>xml-binding-spec</artifactId>
-    <packaging>pom</packaging>
     <version>4.0-SNAPSHOT</version>
-    <name>Jakarta XML Binding Specification</name>
+    <packaging>pom</packaging>
 
-    <properties>
-        <site.output.dir>${project.build.directory}/staging</site.output.dir>
-        <maven.site.skip>true</maven.site.skip>
-        <asciidoctor.maven.plugin.version>2.0.0</asciidoctor.maven.plugin.version>
-        <asciidoctorj.pdf.version>1.5.3</asciidoctorj.pdf.version>
-        <spec.name>jakarta-${project.artifactId}-${project.version}</spec.name>
-        <!-- status: DRAFT, BETA, etc., or blank for final -->
-        <status>DRAFT</status>
-        <maven.build.timestamp.format>MMMM dd, yyyy</maven.build.timestamp.format>
-        <revisiondate>${maven.build.timestamp}</revisiondate>
-    </properties>
+    <name>Jakarta XML Binding Specification</name>
 
     <scm>
         <connection>scm:git:git://github.com/eclipse-ee4j/jaxb-api.git</connection>
         <developerConnection>scm:git:git@github.com:eclipse-ee4j/jaxb-api.git</developerConnection>
         <url>https://github.com/eclipse-ee4j/jaxb-api.git</url>
         <tag>HEAD</tag>
     </scm>
-
     <distributionManagement>
         <site>
             <url>scm:git:git@github.com:eclipse-ee4j/jaxb-api.git</url>
         </site>
     </distributionManagement>
 
+    <properties>
+        <site.output.dir>${project.build.directory}/staging</site.output.dir>
+        <maven.site.skip>true</maven.site.skip>
+        <asciidoctorj.version>2.5.2</asciidoctorj.version>
+        <asciidoctorj.pdf.version>1.6.0</asciidoctorj.pdf.version>
+        <jruby.version>9.3.2.0</jruby.version>
+        <spec.name>jakarta-${project.artifactId}-${project.version}</spec.name>
+        <!-- status: DRAFT, BETA, etc., or blank for final -->
+        <status>DRAFT</status>
+        <maven.build.timestamp.format>MMMM dd, yyyy</maven.build.timestamp.format>
+        <revisiondate>${maven.build.timestamp}</revisiondate>
+    </properties>
```

Comment:
what is the point in moving things around? Are you trying to hide something?

Reply:
It's the pom order according to Maven standards, but practically here so that the spec pom is more similar to that of the other spec projects (and makes it easier to update them if you have to update somewhere over a dozen of them).

Reply:
by "is more similar to that of the other spec projects" you mean those you work on, right? or all?

Reply:
Those mostly overlap ;) As I work on some 27 projects or so with half of them give or take spec projects. But at the same time all as well, as (almost) all come from the original example pom that David provided back in 2018/2019.

Following the Maven conventions is also a good thing, as it makes poms easier digestible by people.

Reply:
so you're saying I should go and blindly change pom formatting in all projects in >30 repos I work on? Most of them are consistent with each other to make this sort of changes easier but only few of them strictly follow conventions

Reply:
Not blindly, but with care. But yes, that would be a good thing. Since the transfer we've all been working on formatting the code according to the Jakarta EE conventions, and indeed, tying up the poms. You can use the tidy plugin for this:

`mvn tidy:pom`

And it will tidy up everything automatically for you.

Reply:
yeah, there is always something one can or could do. But I personally prefer spending time on the other side of the window then doing this sort of things every other year someone comes with "the right set of conventions" and "the right tools" for that, so stick with what is currently there unless there is strong requirement on changing it

Reply:
I think the conventions go back to around 2008 at least and haven't changed since ;)

https://maven.apache.org/developers/conventions/code.html

There's no tools necessary it's just a preferred order of things in the pom.

Reply:
Some projects pre-dates that - so used maven 1 or even were using ant. For instance activation or mail go to the last century, sth like 1997 (or 1999) unless I’m mistaken

Reply:
Moving jaxb-tck out from make based build would be more appreciated